### ISSUE TITLE
Not working on CI [unicorn/expiring-todo-comments]

### ISSUE THREAD

**Issue by cseas** [sindresorhus/eslint-plugin-unicorn#2405]
The expiring-todo-comments config is working when running ESLint on my local system but the failing todo isn't marked as Error when I'm running the same in my GitHub Actions CI.

unicorn/expiring-todo-comments

```js
// ESLint config
{ 
    ...
    'unicorn/expiring-todo-comments': 'error',
}

// Expiring todo comment 
// Todo @cseas [2024-07-20]: Expecting this to fail on CI
```

I ran the `date` command on both my local system and the GitHub Action to try to debug this.
On system: `Mon Jul 22 19:01:03 IST 2024`
On CI: `Mon Jul 22 13:33:06 UTC 2024`

Even with the timezone difference, the todo still should've failed on CI for `2024-07-20`.
